src/index.js

```diff
@@ -30,7 +30,7 @@ const reduceMatchedKeyPaths = (obj, keyPath) => {
 }
 
 const loader = function (content) {
-  if (typeof this.query === 'string') {
+  if (typeof this.query !== 'string') {
```

Comment:
Ok, maybe I am doing something wrong then.  I imported my manifest.json file but when the plugin gets the data, it is a string object.  So you are saying I should use the query object and pass the manifest contents into that object?